commands/help.py

```diff
@@ -0,0 +1,23 @@
+async def response(client, message):
```

Comment:
nit: issue of taste. Response works well for help, but for other commands make less sense. Not sure I have a great replacement, hence the nit.

Reply:
I was thinking as a response to a command. But I can see your concern. Maybe just `command` then? Because then when using it would be `help.command` or `vote.command` etc

Reply:
I like command better